### DISCUSSION TITLE
Splitting test in subpackage

### DISCUSSION THREAD

**Discussion by aTrotier** [MagneticResonanceImaging/MRIReco.jl#89] [Q&A] [answered]
RIght now, the CI fails every time we are modifying the subpackage. @tknopp is running the test on his side before merging.

Is that possible to split the test in the subpackage when possible and running them during CI ?

**Comment by tknopp** [comment]
I don't think so. But the solution is actually to do releases of the sub-package more often.

**Comment by aTrotier** [comment]
Some researchs :

> [Rik Huijzer](https://app.slack.com/team/U01BTNDCUBX)
  [Slack](https://julialang.slack.com/archives/C6A044SQH/p1656415304311119?thread_ts=1656402793.483359&cid=C6A044SQH)
Yes. That is possible, just do Pkg.add(; url="<url>", rev="main") somewhere in your CI. It's also common then to add integration tests to the package that the other packages depend on (e.g., see https://github.com/TuringLang/Libtask.jl/blob/master/.github/workflows/IntegrationTest.yml).

**Comment by aTrotier** [comment, marked as answer]
Solved by PR #98 which use the version of sub-packages in the repository.
